app/controllers/email_authentications/invitations_controller.rb

```diff
@@ -123,4 +125,13 @@ def invite_params
   def temporary_password
     SecureRandom.base64(16)
   end
+
+  def validate_selected_facilities?
+    selected_facilities.blank? && user_params[:access_level] != User.access_levels[:power_user]
```

Comment:
Why use `User.access_levels[:power_user]` instead of just `"power_user"`?

Reply:
Just feel like we shouldn't rely on naked enum references around the code-base, it becomes hard to track down what the source of truth is and what it's actually supposed to be called.

Reply:
Hmm, not sure how much of a difference there is in this usage though, we're just replacing `"power_user"` with `:power_user`. The burden of knowledge is exactly the same. Feel free to ignore this, I realize its pretty nitpicky.